### PR TITLE
auto sync crd resources to chart repo

### PR DESCRIPTION
**Pull request by Haleygo**

### CODE REVIEW

File: .github/workflows/crds.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Update CRDs for chart repo
+
+on:
+  push:
+    branches:
+      - master
```
Comment (f41gh7) [inline .github/workflows/crds.yaml]: WDYT about sync only at release tag?

Comment (Haleygo) [reply .github/workflows/crds.yaml]: Sounds better, updated

**Comment by Haleygo** [comment]
PTAL @Amper @zekker6

File: .github/workflows/crds.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name: Update CRDs for chart repo
+
+on:
+  release:
+    types: [published]
```
Comment (Amper) [inline .github/workflows/crds.yaml:5]: Should we set filter to avoid RC tags here?

Comment (Haleygo) [reply .github/workflows/crds.yaml:5]: We could, but i haven't seen any rc tag, should we filter out with `-rc` suffix?

Comment (Amper) [reply .github/workflows/crds.yaml:5]: Hm... it's weird, i definitely remember that @f41gh7 sometimes releases RC versions to test them in our environments. Maybe he deletes them afterwards.

Comment (Amper) [reply .github/workflows/crds.yaml:5]: https://hub.docker.com/r/victoriametrics/operator/tags?page=&page_size=&ordering=&name=rc

Comment (Haleygo) [reply .github/workflows/crds.yaml:5]: thanks, updated

**Pull request merged**